### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brhv098**
added real racing 3 and google play games to supported list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -342,4 +342,6 @@ wsa://com.android.settings
 | Zenly (w/o GMS) | 4.55.2 | ⚠️ | App crashes after login, but background location works |
 | Zoom | 5.8.3.2634 | ⚠️ | Camera severely glitched, share screen doesn't work due to WSA's windowed nature. | |
 | СберБанк (SberBank) | 12.9.0 | ✅ ||
-| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported |
+| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported ||
+| Real Racing 3 | 10.1.0 | ✅ |no keyboard support, controller works. ||
+| google play games | current | ✅ |works. |
```
Comment (riverar) [inline README.md:347]: Not sure what this line means. Is this an actual app name?

File: README.md [hunk 2]
```diff
@@ -342,4 +342,6 @@ wsa://com.android.settings
 | Zenly (w/o GMS) | 4.55.2 | ⚠️ | App crashes after login, but background location works |
 | Zoom | 5.8.3.2634 | ⚠️ | Camera severely glitched, share screen doesn't work due to WSA's windowed nature. | |
 | СберБанк (SberBank) | 12.9.0 | ✅ ||
-| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported |
+| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported ||
+| Real Racing 3 | 10.1.0 | ✅ |no keyboard support, controller works. ||
```
Comment (riverar) [inline README.md:346]: Formatting/grammar: Add space after `|` and capitalize the `N`.

**Review by riverar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pratyakshm) [reply README.md:347]: If you are asking about Google Play Games, yes that is an actual app name. 

[Google Play Games - Apps on Google Play](https://play.google.com/store/apps/details?id=com.google.android.play.games)

Comment (riverar) [reply README.md:347]: Ah, funny. We'll need a version number though @brhv098 or delete this.

File: README.md [hunk 3]
```diff
@@ -342,4 +342,6 @@ wsa://com.android.settings
 | Zenly (w/o GMS) | 4.55.2 | ⚠️ | App crashes after login, but background location works |
 | Zoom | 5.8.3.2634 | ⚠️ | Camera severely glitched, share screen doesn't work due to WSA's windowed nature. | |
 | СберБанк (SberBank) | 12.9.0 | ✅ ||
-| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported |
+| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported ||
```
Comment (pratyakshm) [inline README.md:345]:
```suggestion
| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 is unsupported ||
```

File: README.md (hunk 2 above)
Comment (pratyakshm) [inline README.md:346]:
```suggestion
| Real Racing 3 | 10.1.0 | ✅ | Only controller is supported. keyboard doesn't work ||
```

File: README.md (hunk 1 above)
Comment (pratyakshm) [inline README.md:347]:
```suggestion
```

**Pull request merged**